deconz/config.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "deCONZ",
+    "version": "2.05.47",
+    "slug": "deconz",
+    "description": "Control a ZigBee network with Conbee or RaspBee by Dresden Elektronik",
+    "arch": ["aarch64","amd64","armhf","i386"],
+    "url": "https://home-assistant.io/addons/deconz",
+    "startup": "system",
+    "boot": "auto",
+    "webui": "http://[HOST]:[PORT:8080]",
+    "host_network": true,
```

Comment:
Why did we need to run on the host network?

Reply:
The deCONZ web ui (Phoscon) uses uPNP, so for correct operation I've had to allow the container to run on the host network.